static/src/stylesheets/amp/_hosted.scss

```diff
@@ -11,7 +11,7 @@ $header-height: 48px;
     text-align: right;
 }
 
-.hosted__glogo svg {
+.hosted__glogo .inline-logo__svg {
```

Comment:
Yeah, we're not passing very specific class names through to the svg so playing it safe here. Though I suppose when we start only loading this css in hosted pages it will be fine